__tests__/app.test.js

```diff
@@ -104,3 +105,25 @@ describe("ERROR: GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("status: 200, should responds with an article object, containing the properties author, title, article_id, body, topic, created_at, votes, article_img_url and comment_count", () => {
+    return request(app)
+      .get(`/api/articles`)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Array);
+        expect(body).toBeSorted({ descending: true });
```

Comment:
This doesn't look quite right, you should take a look at the docs for jest-sorted here...

https://www.npmjs.com/package/jest-sorted?activeTab=readme

The example when checking the sort order of an array of objects:


`expect([{ id: 1 }, { id: 2 }, { id: 3 }]).toBeSortedBy('id');`